goss.yaml

```diff
@@ -0,0 +1,16 @@
+command:
+  self-tests-health:
+    exec: juju ssh loki-k8s/0 /charm/bin/pebble health
```

Comment:
- Is the idea to have a `goss.yaml` per charm?
- Can we loop in goss over several charm names, if the checks are the same?

Reply:
In the case of a bundle we could have 1 `goss.yaml` which is responsible for checking each charm's pebble health and running some extra external checks like relations.

Technically there is no need to have the `goss.yaml` in the charm repo (these should be in a COS Doctor repo).

Reply:
You can loop over a list of variables defined in a `vars.yaml` like so:

```
# Loop over users
user:
{{range .Vars.users}}
  {{.}}:
    exists: true
    groups:
    - {{.}}
    home: /home/{{.}}
    shell: /bin/bash
{{end}}
```